docker/entrypoint.py

```diff
@@ -63,12 +67,7 @@ def change_tz():
   # TZ is not symlinked and defined TZ exists
   if tz not in os.readlink('/etc/localtime') and os.path.isfile('/usr/share/zoneinfo/' + tz):
     loggersrv.info("Setting timzeone to %s" % tz )
-    os.remove('/etc/localtime')
-    os.symlink(os.path.join('/usr/share/zoneinfo/', tz), '/etc/localtime')
-    f = open("/etc/timezone", "w")
-    f.write(tz)
-    f.close()
-
+    time.tzset()
```

Comment:
Cool addition, but does it work cross-scripts? After all, this is the entrypoint, executed as root - `py-kms` itself will later run even as an other user!

Reply:
you are right, not available for windows, will add a check for function's availability. According to documentation when set, all time libraries should act accordingly: https://docs.python.org/3/library/time.html

Reply:
Oh, sorry - you've got that wrong. It does work only on Linux (of course - we are running inside a Linux container anyways). I'm asking, because this command is executed inside the `entrypoint.py`, which then hands off the `py-kms` server using console commands (NOT a module import). Does the changes persist between the hand-off? Have you ever tested that?